mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/NavAnsattService.kt

```diff
@@ -29,6 +32,35 @@ class NavAnsattService(
         return navAnsattRepository.getAll(roller = filter.roller)
     }
 
+    suspend fun addUserToKontaktpersoner(navIdent: NavIdent, tx: Session) {
+        val kontaktPersonGruppeId = roles.find { it.rolle == NavAnsattRolle.KONTAKTPERSON }?.adGruppeId
+        requireNotNull(kontaktPersonGruppeId)
+
+        val ansatt = navAnsattRepository.getByNavIdent(navIdent)
+            ?: microsoftGraphService.getNavAnsattByNavIdent(navIdent, AccessType.M2M)?.let {
+                NavAnsattDto.fromAzureAdNavAnsatt(it, roller = emptySet())
+            }
+        requireNotNull(ansatt) {
+            "Fant ikke ansatt med navIdent=$navIdent i AzureAd"
+        }
+
+        if (ansatt.roller.contains(NavAnsattRolle.KONTAKTPERSON)) {
+            return
+        }
+
+        navAnsattRepository.upsert(
+            NavAnsattDbo.fromNavAnsattDto(ansatt).copy(roller = ansatt.roller.plus(NavAnsattRolle.KONTAKTPERSON)),
```

Comment:
Dette vil potensielt kunne overskrive "skalSlettesDato", altså sette denne til `null` igjen om den allerede har blitt satt?

Reply:
Sikkert ikke noe krise, er vel samme oppførsel som `getOrSynchronizeNavAnsatt`. Den får vel bare en ny slettedato dagen etter i så fall, så virker veldig edge case

Reply:
Hmm, jeg tror ikke det for hvis den var i db så er det den som blir brukt. Henter bare fra ms når den ikke finnes i db